packages/composite-checkout/src/lib/form-status.ts

```diff
@@ -1,26 +1,32 @@
 /**
  * External dependencies
  */
-import { useReducer, useMemo, useCallback, useContext, useEffect } from 'react';
+import { useCallback, useContext, useEffect, useMemo, useReducer } from 'react';
```

Comment:
These clean-ups are fine, but to aid in review it would be helpful to have them be in their own commit. That way it's clear they weren't functional changes.